src/components/AudioPlayer/Player/Player.component.js

```diff
@@ -3,18 +3,19 @@ import propTypes from 'prop-types';
 
 import {
   Repeat,
-  Rewind,
+  RewindButtonContainer,
   PlayContainer,
-  Forward, Shuffle,
+  ForwardButtonContainer,
+  Shuffle,
 } from '../PlayerButtons';
 
 const PlayerComponent = ({ isDisabled, audioRef }) => (
   <div className={`buttons ${isDisabled ? 'disable-buttons' : ''}`}>
     <div className="buttons-container">
       <Repeat />
-      <Rewind />
+      <RewindButtonContainer />
```

Comment:
I don't think we want the `container` part in the name here. Remember `component/container` it's just the way we abstract business logic rather than a naming convention.